include/netlink/route/neighbour.h

```diff
@@ -79,6 +79,9 @@ extern int			rtnl_neigh_get_vlan(struct rtnl_neigh *);
 extern void			rtnl_neigh_set_master(struct rtnl_neigh *, int);
 extern int			rtnl_neigh_get_master(struct rtnl_neigh *);
 
+extern void			rtnl_neigh_set_nhid(struct rtnl_neigh *, unsigned int);
```

Comment:
does the patch lack a
```c
    if (tmpl->ce_mask & NEIGH_ATTR_NHID)
        NLA_PUT_U32(msg, NDA_NH_ID, tmpl->n_nhid);
```

?

otherwise, there is no point of having `rtnl_neigh_set_nhid()` in the public API.

Reply:
I've added population the attribute.